lte/gateway/fabfile.py

```diff
@@ -304,7 +304,8 @@ def federated_integ_test(
     execute(_make_integ_tests)
     sleep(20)
     execute(run_integ_tests, "federated_tests/s1aptests/test_attach_detach.py")
-
+    sleep(20) #ToDo: Remove individual test run and add in defs.mk
+    execute(run_integ_tests, "federated_tests/s1aptests/test_attach_detach_multi_ue.py")
```

Comment:
Why does it have to run individually? Can t we do the same as LTE integ test where the test are run out of a list?

Reply:
hi @uri200 
It is addressed now. I have created separate make option `fed_integ_test`.  Is it okay  ?